examples/kafka-hub/hub/modules/connections/connections.bal

```diff
@@ -42,7 +64,18 @@ public isolated function createMessageConsumer(string topicName, string groupNam
     kafka:ConsumerConfiguration consumerConfiguration = {
         groupId: groupName,
         topics: [topicName],
-        autoCommit: false
+        autoCommit: false,
+        secureSocket: {
+            cert: "./resources/brokercerts/broker.public.crt",
```

Comment:
shall we use a define a separate securesocket record and use it everywhere?

Reply:
Fixed with [6724bd54d5de376d9b51a29dd7869d174541b562](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/877/commits/6724bd54d5de376d9b51a29dd7869d174541b562)